config/import.plugin.js

```diff
@@ -58,7 +58,7 @@ ImportPlugin.prototype.apply = function (compiler) {
             const indexOfExports = originalSource.indexOf("exports.", indexOfMethod);
             const indexOfDelimeter = originalSource.indexOf(";", indexOfExports);
 
-            const codeToInject = `exports.metadata = { version: ${JSON.stringify(require("../package.json").devDependencies["progress-sitefinity-adminapp-sdk"])}, name: "${constants.extensionsKey}" };`;
+            const codeToInject = `arguments[1].metadata = { version: ${JSON.stringify(require("../package.json").devDependencies["progress-sitefinity-adminapp-sdk"])}, name: "${constants.extensionsKey}" };`;
```

Comment:
Can you please explain?

Reply:
Yes, when the source code is minified, the exports keyword is not available. To ensure that this works both with minified and unminified code I read the second argument